lib/places_alloverse_com/accounts/credential.ex

```diff
@@ -3,8 +3,12 @@ defmodule PlacesAlloverseCom.Accounts.Credential do
   import Ecto.Changeset
   alias PlacesAlloverseCom.Accounts.User
 
+  # @derive {Inspect, except: [:password]}
```

Comment:
debugging junk? or is this line good for making sure that passwords are never printed to logs accidentally and SHOULD be uncommented and kept in?